cards/cards.css

```diff
@@ -0,0 +1,76 @@
+.disp-card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card-col {
+  display: flex;
+  flex-direction: column;
+  width: 30rem;
+  border: 1px solid black;
+  border-radius: 0.3rem;
+  box-shadow: 1px 1px;
+  margin: 1rem;
+}
+
+.card-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30rem;
+  height: 15rem;
+  max-width: 100%;
```

Comment:
Do use object-fit:cover for images, It's a good practise.Also I don't think flex is needed here

Reply:
Sure, Got it. Will change it. Thankyou :smile:

Reply:
@poojavishnoi Anyways, I'm doing `object-fit: cover` here as I missed on it but due to some reason that is blurring the image a bit may be due to forceful fitting of the image in the container. Any solution for that?